tests/scripts/create-dev-cluster.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
```

Comment:
This should go below the copyright message.

Reply:
@sp98  wrote:

> This should go below the copyright message.

done.  Moved.


But in general I would rather remove the boilerplate copyright comments everywhere: they are redundant and serve no purpose

Reply:
I see 2 instances of this now, so one needs removed. And I also agree that the copyright note is a bit much, especially for a script. Feel free to remove it.

Reply:
@BlaineEXE  wrote:
> I see 2 instances of this now, so one needs removed. And I also agree that the copyright note is a bit much, especially for a script. Feel free to remove it.

Thanks for catching this! I removed both the duplication and the boilerplate comment.